### ISSUE TITLE
Bug: failing tests

### ISSUE THREAD

**Issue by a0m0rajab** [unifyai/ivy#21211]
### Bug Explanation

The tests in the repo have been failing for a couple of months. Even an empty commit would have the test to fail, it would be great if we had an opportunity to solve the tests slowly, What I am thinking about is doing the next: 

- trigger the CI/CD tests with an empty commit
- check the failing tests to document them and solve them 
- close all tasks related to tests, it's making it feels that there are a lot of unsolved issues in the repo but it's not! 

As for today, the number of tasks opened for failing tests is:  3093
![image](https://github.com/unifyai/ivy/assets/18273833/73643fc3-6b25-4c6e-a291-3617641b8d41)
https://github.com/unifyai/ivy/issues?q=is%3Aissue+is%3Aopen+test+label%3A%22Failing+Test%22

The number of the whole tasks is: 3717 

That leaves us with: 624 tasks 
420 of them are just subtasks (function task) 

https://github.com/unifyai/ivy/issues?q=is%3Aissue+is%3Aopen+label%3A%22Sub+Task%22+-label%3A%22Failing+Test%22

### Steps to Reproduce Bug

N/A

### Environment

N/A

### Ivy Version

N/A

### Backend

- [ ] NumPy
- [ ] TensorFlow
- [ ] PyTorch
- [ ] JAX

### Device

N/A

**Comment by a0m0rajab** [comment]
The bugs failing tests were resolved, not sure how, that's why we close this issue right now.
